.github/workflows/upstream.yml

```diff
@@ -0,0 +1,20 @@
+name: upstream
+
+on:
+  schedule:
+    - cron: '30 7,19 * * *'
+  workflow_dispatch:
+
+jobs:
+  versioncheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
```

Comment:
Please could we have commit hashes rather than versions for these actions (cf. `coverage.yml`)